### ISSUE TITLE
Add this code

### ISSUE THREAD

**Issue by sdsunjay** [sdsunjay/nlp_product_reviews#5]
import torch
from torch import nn
from torch.utils.data import TensorDataset, DataLoader
from transformers import BertTokenizer, BertModel
import nltk
from nltk.corpus import stopwords
nltk.download('stopwords')
nltk.download('punkt')

# Load the BERT model
bert_model = BertModel.from_pretrained('bert-base-uncased')

# Initialize the BERT tokenizer
tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')

# Load NLTK's English stop words
stop_words = set(stopwords.words('english'))

# Create lists to store the embeddings and labels
embeddings = []
labels = []

# Open the file containing the sentences
with open('sentences.txt', 'r') as file:
    # Iterate through each line (sentence) in the file
    for line in file:
        # Tokenize the sentence
        tokens = nltk.word_tokenize(line)
        # Remove stop words from the sentence
        tokens = [token for token in tokens if token.lower() not in stop_words]
        # Rejoin the cleaned tokens into a single sentence
        cleaned_sentence = ' '.join(tokens)
        # Tokenize the cleaned sentence
        tokens = tokenizer.tokenize(cleaned_sentence)
        
        # Convert the tokens to their corresponding IDs
        tokens_ids = tokenizer.convert_tokens_to_ids(tokens)
        
        # Convert the tokens IDs to a PyTorch tensor
        tokens_tensor = torch.tensor([tokens_ids])
        
        # Pass the tokens tensor through the BERT model to get the word embeddings
        _, word_embeddings = bert_model(tokens_tensor)
        
        # Append the word embeddings to the list of embeddings
        embeddings.append(word_embeddings)
        
        # Append the label to the list of labels
        if "bad" in line:
            labels.append(1)
        else:
            labels.append(0)

# Convert the lists to PyTorch tensors
embeddings_tensor = torch.stack(embeddings)
labels_tensor = torch.tensor(labels)

# Create a TensorDataset from the tensors
dataset = TensorDataset(embeddings_tensor, labels_tensor)

# Create a DataLoader from the dataset
dataloader = DataLoader(dataset, batch_size=32, shuffle=True)

# Define the model
class TextClassificationModel(nn.Module):
    def __init__(self, embedding_size, hidden_size, num_classes):
        super(TextClassificationModel, self).__init__()
        self.linear1 = nn.Linear(embedding_size, hidden_size)
        self.linear2 = nn.Linear(hidden_size, num_classes)

    def forward(self, x):
        x = torch.relu(self.linear1(x))
        x = self.linear2(x)
        return x

# Initialize the model
embedding_size = word_embeddings.shape[-1]
hidden_size = 256
num_classes = 2
model = TextClassificationModel(embedding_size, hidden_size, num_classes)

# Define the loss function and optimizer
criterion = nn.CrossEntropyLoss()
optimizer = torch.optim.Adam(model.parameters())

# Train the model
num_epochs = 10
for epoch in range(num_epochs):
    for i, (embeddings, labels) in enumerate(dataloader):
        # Forward pass
        outputs = model(embeddings)
        loss = criterion(outputs, labels)

        # Backward and optimize
        optimizer.zero_grad()
        loss.backward()
        optimizer.step()

    print ('Epoch [{}/{}], Loss: {:.4f}'.format(epoch+1, num_epochs, loss.item()))

# Test the model
with torch.no_grad():
    correct = 0
    total = 0
    for embeddings, labels in dataloader:
        outputs = model(embeddings)
        _, predicted = torch.max(outputs.data, 1)
        total += labels.size(0)
        correct += (predicted == labels).sum().item()

    print('Accuracy of the model on the test data: {} %'.format(100 * correct / total))
